{{cookiecutter.project_slug}}/.pre-commit-config.yaml

```diff
@@ -0,0 +1,32 @@
+exclude: .+/migrations/.+\.py
+default_stages: [commit]
+fail_fast: true
+
+repos:
+-   repo: https://github.com/pre-commit/pre-commit-hooks
+    rev: master
+    hooks:
+      - id: trailing-whitespace
+        files: (^|/){{cookiecutter.project_slug}}/.+\.(py|html|sh|css|js)$
+
+-   repo: local
+    hooks:
+      - id: black
+        name: black
+        entry: black
+        args: [--line-length=80]
+        language: python
+        types: [python]
+        files: (^|/){{cookiecutter.project_slug}}/.+\.py$
+        verbose: true
```

Comment:
Looking at this:

> forces the output of the hook to be printed even when the hook passes

Seems a bit much, can we remove it?

```suggestion
```

Reply:
Yes, I agree to remove it.